sqre-uservice-{{cookiecutter.svc_name}}/uwsgi.ini

```diff
@@ -0,0 +1,9 @@
+[uwsgi]
+; If VIRTUAL_ENV is set then use its value to specify the virtualenv directory
+if-env = VIRTUAL_ENV
+virtualenv = %(_)
```

Comment:
If VIRTUAL_ENV is in the environment, then the uwsgi virtualenv argument is set to that value.  Otherwise it runs in the system context, which is usually not what you want.